Sources/ProjectDescription/SettingsTransformers.swift

```diff
@@ -0,0 +1,147 @@
+import Foundation
+
+extension Dictionary {
+    mutating func merge(with dict: [Key: Value]) {
+        dict.forEach { self[$0.key] = $0.value }
+    }
+
+    func merging(with dict: [Key: Value]) -> Dictionary {
+        var copy = self
+        copy.merge(with: dict)
+        return copy
+    }
+}
+
+private extension Bool {
```

Comment:
I am not sure how necessary `asSettingValue` is? I don't really like the fact that you can do `SettingValue(stringLiteral: string)` and `string.asSettingValue`.
But what I'd definitely do, and I think you are hitting a fine point here, is extend `SettingValue` with `ExpressibleByBooleanLiteral`

Reply:
@fortmarek  It's not really necessary, it's just that it reads a bit better in my opinion. I can change it, but I personally prefer the shorter `string.asSettingValue`.

Reply:
@fortmarek `ExpressibleByBooleanLiteral` was a great idea! I removed the `asSettingValue` properties but added private inits to keep them short (without the need to pass the labels) 👍

Reply:
Great job, I like that 👍